ICSharpCode.CodeConverter/Shared/DocumentExtensions.cs

```diff
@@ -113,5 +116,11 @@ private static SyntaxNode WithWarningAnnotation(SyntaxNode node, string warningT
         {
             return node.WithAdditionalAnnotations(new SyntaxAnnotation(AnnotationConstants.ConversionErrorAnnotationKind, warningText));
         }
+        private static OptionSet GetVBOptions(DocumentOptionSet options) {
+            return options.WithChangedOption(SimplificationOptions.PreferImplicitTypeInLocalDeclaration, false);
```

Comment:
For some reason I couldn't find the way to change the simplification options when I looked, so this is extra awesome! Now I can go and change a bunch of others that cause issues 😁